scripts/travis-emscripten/build_emscripten.sh

```diff
@@ -53,7 +53,7 @@ cmake \
   -DBoost_USE_STATIC_RUNTIME=1 \
   -DTESTS=0 \
   ..
-make -j 4 soljson
+make -j "${NJOBS:-4}" soljson
```

Comment:
Maybe you should also modify Travis config to set this variable? Or is it just that you're fine with the default and don't want to change it to anything higher right now?